api/client/client.go

```diff
@@ -997,3 +997,51 @@ func (c *Client) IsMFARequired(ctx context.Context, req *proto.IsMFARequiredRequ
 	}
 	return resp, nil
 }
+
+// GetTrustedCluster returns a Trusted Cluster by name.
+func (c *Client) GetTrustedCluster(ctx context.Context, name string) (types.TrustedCluster, error) {
+	if name == "" {
+		return nil, trace.BadParameter("missing name")
```

Comment:
Nit: Prefer error messages that provide a bit more context (e.g. `missing trusted cluster name`, or even `cannot get trusted cluster, missing name`).